metadata-suggestions.md

```diff
@@ -0,0 +1,682 @@
+
+### Adding metadata
+
+With the `add` operation, the live import suggests to add metadata new values.
+
+Example response, suggestion the addition of three values:
+
+```json
+{
+  "dc.subject" : {
+    "currentvalues" : [],
+    "suggestions": [
+      {
+        "operations": [ "add/metadata/dc.subject/-" ],
+        "newvalue": "DNA replication inhibitors"
+      },
+      {
+        "operations": [ "add/metadata/dc.subject/-" ],
+        "newvalue": "Mycobacterium"
+      }
+    ]
+  }
+}
+```
+
+If this patch is applied hereafter, the new metadata state is:
+
+```json
+{
+  "metadata": {
+    "dc.contributor.author": [
+      { "value": "D. Pollard", "language": null, "authority": null, "confidence": -1 },
+      { "value": "C. E. Wylie", "language": null, "authority": "virtual::1373", "confidence": 600 }
+    ],
+    "dc.date.issued": [
+      { "value": "2019", "language": null, "authority": null, "confidence": -1 }
+    ],
+    "dc.subject": [
+      { "value": "DNA replication inhibitors", "language": null, "authority": null, "confidence": -1 },
+      { "value": "Mycobacterium", "language": null, "authority": null, "confidence": -1 }
+    ]
+  }
+}
+```
+
+### Replacing metadata
+
+With `replace`, the live import suggests to overwrite a single existing metadata value:
+* `replace/<:place>` is used to suggest replacing the value at <:place> with the new value
+
+Example response, replacing the the first `dc.date.issued`:
+
+```json
+{
+    "dc.date.issued" : {
+      "currentvalues": [ "2019" ],
+      "suggestions": [
+        {
+          "operations": [ "replace/metadata/dc.date.issued/0" ],
+          "newvalue": "2019-09-23"
+        }
+      ]
+    }
+}
+```
+
+If this patch is applied hereafter, the new metadata state is:
+
+```json
+{
+  "metadata": {
+    "dc.contributor.author": [
+      { "value": "D. Pollard", "language": null, "authority": null, "confidence": -1 },
+      { "value": "C. E. Wylie", "language": null, "authority": "virtual::1373", "confidence": 600 }
+    ],
+    "dc.date.issued": [
+      { "value": "2019-09-23", "language": null, "authority": null, "confidence": -1 }
+    ],
+    "dc.subject": [
+      { "value": "DNA replication inhibitors", "language": null, "authority": null, "confidence": -1 },
+      { "value": "Mycobacterium", "language": null, "authority": null, "confidence": -1 }
+    ]
+  }
+}
+```
+
+### Replacing name variants
+
+With `namevariant`, the live import suggests to overwrite a single name variant:
+* `namevariant/<:place>` is used to suggest replacing the name variant of the relationship at <:place> with the new value
+
+Example response, replacing the the second `dc.contributor.author` name variant:
+
+```json
+{
+    "dc.contributor.author" : {
+      "currentvalues" : [ "D. Pollard", "C. E. Wylie[virtual::1373]" ],
+      "suggestions" : [
+        {
+          "operations": [ "namevariant/metadata/dc.contributor.author/1", "add/metadata/dc.contributor.author/-" ],
```

Comment:
Good example of having multiple options for "operations".  We might want to point that out in the description below...that in this scenario two operations were suggested,  and the user chose the *first* operation to replace the "namevariant"

Reply:
This was already mentioned in the section "Changes suggested from the live import", but I've also mentioned this in the section above to further clarify it

Reply:
I think you misunderstood my minor comment (I realize it wasn't clear).  I was referring to slightly tweaking line 595 of this file which currently just says:
> If this patch is applied hereafter, the new metadata state is:

I think we might want to point out that there are *two possible operations* in the example, and that the first one is "chosen" by the user / client.  So, maybe changing it to say something like:
> The above example includes two possible operations: an option to replace the namevariant, and an option to add the `newvalue` as an additional author. Assuming the client chooses the first operation (replace the namevariant), the new metadata state is:

Reply:
Thanks for the clarification, this is indeed much clearer